lib/handlers/hard-quote/handler.ts

```diff
@@ -51,25 +51,26 @@ export class QuoteHandler extends APIGLambdaHandler<
 
     // Instead of decoding the order, we rely on frontend passing in the requestId
     //   from indicative quote
+    const logBody = {
+      requestId: request.requestId,
+      quoteId: request.quoteId,
+      tokenInChainId: request.tokenInChainId,
+      tokenOutChainId: request.tokenOutChainId,
+      tokenIn: request.tokenIn,
+      tokenOut: request.tokenOut,
+      offerer: request.swapper,
+      amount: request.amount.toString(),
+      type: TradeType[request.type],
+      numOutputs: request.numOutputs,
+      cosigner: request.order.info.cosigner,
+      createdAt: timestampInMstoSeconds(start),
+      createdAtMs: start.toString(),
+    }
     log.info({
       eventType: 'HardRequest',
-      body: {
```

Comment:
What's the reason for moving this outside?

Reply:
It was to use the same object for logging the response. But no longer relevant with the latest changes.